script.js

```diff
@@ -1,4 +1,54 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // Creating a variable for a user input of scissors, paper or stone
+  var userChoice = input;
+
+  // Creating a computer random choice through a system similar to rolling dice
+  var computerSelectNum = Math.floor(Math.random() * 3) + 1;
+
+  // Assigning the random number of 1 to 3 with scissors, paper and stone respectively
+  var computerChoice = "default";
+  if (computerSelectNum == 1) {
+    computerChoice = "scissors";
+  } else if (computerSelectNum == 2) {
+    computerChoice = "paper";
+  } else if (computerSelectNum == 3) {
+    computerChoice = "stone";
+  }
+
+  // Writing a function that would produce who wins or loses through the game rules
+  var SPS = function (userChoice) {
```

Comment:
Personally, I would call the function gamePlay() instead of SPS for clearer naming of variables.